pytest/lib/branches.py

```diff
@@ -0,0 +1,27 @@
+
+import os
+import subprocess
+
+
+def current_branch():
+    return subprocess.check_output([
+        "git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
+
+
+def compile_binary(branch):
+    """For given branch, compile binary."""
+    # TODO: download pre-compiled binary from github for beta/stable?
+    prev_branch = current_branch()
+    subprocess.call(['git', 'checkout', branch])
```

Comment:
It's possible cargo.lock changed, so I think a stash and a git stash pop after checkout prev_branch below will be more robust